autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -453,6 +455,7 @@ def on_node_executor_stop(cls):
             return  # already shutting down
 
         logger.info(f"[on_node_executor_stop {cls.pid}] ⏳ Disconnecting DB...")
+        redis.disconnect()
```

Comment:
please move this below the `db.disconnect()`, for 2 reasons:
- it's now directly after a log statement that says "Disconnecting DB"
- startup/shutdown sequences should be each other's reverse

Reply:
Updated